### PR TITLE
HW1, HW2 and HW3 added

### PR DESCRIPTION
**Pull request by avu-ua**

### CODE REVIEW

File: Anishchenko_V/HW3/task_2.py [hunk 1]
```diff
@@ -0,0 +1,26 @@
+# Take input number from user
```
Comment (kolyasalubov) [inline Anishchenko_V/HW3/task_2.py:1]: excellent work

File: Anishchenko_V/HW3/task_3.py [hunk 2]
```diff
@@ -0,0 +1,7 @@
+# Interchange the values of two variables without using the third variable
+
```
Comment (kolyasalubov) [inline Anishchenko_V/HW3/task_3.py:2]: good job

File: Anishchenko_V/HW4/Celsius.py [hunk 3]
```diff
@@ -0,0 +1,13 @@
+while True:
```
Comment (kolyasalubov) [inline Anishchenko_V/HW4/Celsius.py:1]: good job

File: Anishchenko_V/HW3/task_1.py [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+12.In the face of ambiguity, refuse the temptation to guess.
+13.There should be one-- and preferably only one --obvious way to do it.
+14.Although that way may not be obvious at first unless you're Dutch.
+15.Now is better than never.
+16.Although never is often better than *right* now.
+17.If the implementation is hard to explain, it's a bad idea.
+18.If the implementation is easy to explain, it may be a good idea.
+19.Namespaces are one honking great idea -- let's do more of those!'''
+
+
```
Comment (kolyasalubov) [inline Anishchenko_V/HW3/task_1.py:21]: good job

File: Anishchenko_V/HW5_loops/task1.py [hunk 5]
```diff
@@ -0,0 +1,13 @@
+numbers_list = [15, 2, -3, 41, 187, 91, -214, 0, 1, 16]
```
Comment (kolyasalubov) [inline Anishchenko_V/HW5_loops/task1.py:1]: good job

File: Anishchenko_V/HW5_loops/task2.py [hunk 6]
```diff
@@ -0,0 +1,12 @@
+user_number = int(
```
Comment (kolyasalubov) [inline Anishchenko_V/HW5_loops/task2.py:1]: good job
look at this implementation:
user_number = int(input('Enter a natural number (i.e. a whole number that is greater than 0): '))

fibonacci_list = [0, 1]

while True:
    next_number = fibonacci_list[-1] + fibonacci_list[-2]
    if next_number > user_number:
        break
    fibonacci_list.append(next_number)

print(f'List of Fibonacci numbers up to your number ({user_number}):')
print(fibonacci_list)

File: Anishchenko_V/HW5_loops/task3_optional_bonus.py [hunk 7]
```diff
@@ -0,0 +1,9 @@
+user_number = int(input('Enter a whole number: '))
```
Comment (kolyasalubov) [inline Anishchenko_V/HW5_loops/task3_optional_bonus.py:1]: good job
look at this implementation:

user_number = int(input('Enter a whole number: '))

if user_number < 0:
    print("Factorial is not defined for negative numbers.")
elif user_number == 0:
    print("0! = 1")
else:
    factorial = 1
    for i in range(1, user_number + 1):
        factorial *= i
    print(f'{user_number}! = {factorial}.')

File: Anishchenko_V/HW5_loops/task2.py (hunk 6 above)
Comment (avu-ua) [reply Anishchenko_V/HW5_loops/task2.py:1]: Thank you. The alternative implementation is noted, the only my comment here is that for a specific case when user_number = 1 my implementation return the sequence of [0, 1] whilst the alternative one - [0, 1, 1].
I am not sure which outcome for this specific case is correct / as required by the terms of the task.

Comment (kolyasalubov) [reply Anishchenko_V/HW5_loops/task2.py:1]: you are right, according to the condition it is more correct for the program to return [0,1]

File: Anishchenko_V/HW6_collections/task1_divisibles.py [hunk 8]
```diff
@@ -0,0 +1,10 @@
+even = [i for i in range(2, 11, 2)]
```
Comment (kolyasalubov) [inline Anishchenko_V/HW6_collections/task1_divisibles.py:1]: good job

File: Anishchenko_V/HW6_collections/task2_check_login.py [hunk 9]
```diff
@@ -0,0 +1,6 @@
+login = input('Enter login: ')
```
Comment (kolyasalubov) [inline Anishchenko_V/HW6_collections/task2_check_login.py:1]: good job
look at this way

user_name = input('Hello, please input your Log in:')

while user_name != 'First':
    user_name = input('Error: wrong username, please try one more time. Username:')
else:
    print('Greeting. Access granted!!!', user_name)

**Pull request merged**